package/utils/xfsprogs/Makefile

```diff
@@ -8,24 +8,25 @@
 include $(TOPDIR)/rules.mk
```

Comment:
Please extend the commit description like @aparcar already requested when you send this to the mailing list.

I would prefer to apply this already before the other changes.

Reply:
Done